<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trackerblocker</class>
 <widget class="QWidget" name="trackerblocker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="BlockTrackers_checkBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Block Trackers</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>238</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="NumberTrackerBlockerRules_groupBox">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox {
     border: 2px solid gray;
     border-radius: 5px;
/*    margin-top: 1ex;*/ /* leave space at the top for the title */
 }

 QGroupBox::title {
background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
/*     padding:2 13px;*/
    padding: 3 3px;
/*     subcontrol-origin: margin;*/
 }</string>
         </property>
         <property name="title">
          <string>Number of Tracker Blocker Rules Loaded: </string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="TrackerBlockerRules_treeWidget">
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>Rule</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="TrackerBlockerImportRules_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Import Rules</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="TrackerBlockerExportRules_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Export Rules</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="TrackerBlockerSave_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Save Changes</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>68</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Tracker Blocker File:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="TrackerBlockerFile_lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="TrackerBlockerFile_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Browse ...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="BlockedTrackers_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 2px solid gray;
     border-radius: 5px;
/*    margin-top: 1ex;*/ /* leave space at the top for the title */
 }

 QGroupBox::title {
background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
/*     padding:2 13px;*/
    padding: 3 3px;
/*     subcontrol-origin: margin;*/
 }</string>
      </property>
      <property name="title">
       <string>Blocked Trackers</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="topMargin">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="TrackerBlocker_stackedWidget">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
